apps/web/src/app/[locale]/(dashboard)/[wsId]/structure.tsx

```
@@ -138,217 +129,137 @@ export function Structure({
+      navCollapsedSize={navCollapsedSize}
+      isCollapsed={isCollapsed}
+      setIsCollapsed={setIsCollapsed}
+      debouncedSaveSizes={debouncedSaveSizes}
+      debouncedSaveCollapsed={debouncedSaveCollapsed}
+      header={header}
+      mobileHeader={mobileHeader}
+      sidebarHeader={sidebarHeader}
+      sidebarContent={sidebarContent}
+      actions={actions}
+      userPopover={userPopover}
+      children={children}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)